src/Drivers/AbstractSMS.php

```diff
@@ -81,5 +83,11 @@ public function setDebug($debug)
         $this->debug = $debug;
     }
 
-
+    /**
+     * @param OutgoingMessage $outgoingMessage
+     */
+    protected function dispatchOutgoingMessageSentEvent(OutgoingMessage $outgoingMessage)
+    {
+        event(new SMSSentEvent($outgoingMessage));
```

Comment:
The event dispatcher is going to need to be injected into the constructor and passed down in order for this to be used outside of Laravel.  This will also need to be updated in the ServiceProvider.